build.template

```diff
@@ -1,7 +1,112 @@
-//! Configuration settings for this Rhai build
+//! Fixed hashing seeds for stable hashing.
 //!
-//! This file is auto-generated from `build.template`
+//! Set to [`None`] to disable stable hashing.
+//!
+//! See [`set_rhai_ahash_seed`] for more.
+//!
+//! Alternatively, You can also set this at compile time by disabling `stable_hash` and setting the `RHAI_AHASH_SEED`
+//! environment variable instead.
+//!
+//! E.g. `env RHAI_AHASH_SEED ="[236,800,954,213]"`
+// [236,800,954,213], haha funny yume nikki reference epic uboachan face numberworld nexus moment 100
```

Comment:
Wonder what yume nikki reference is...  :-)

Reply:
its 4 of the numbers that are on the ground in number world

Reply:
Ah, after some Googling, I got it now!